swift/ql/lib/change-notes/2023-09-14-for-in-data-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+
+* The data flow library now supports flow to the loop variable of for-in loops.
```

Comment:
Should we mention that `ForEachStmt.getSequence()` has been removed in favour of `ForEachStmt.getIteratorVar` and `ForEachStmt.getNextCall`, as these things are in principle available to users?  Ideally the former would go through deprecation, though I don't have strong feelings about this - I highly doubt anyone's using it directly.

Reply:
I've added `getSequence` back in to `codeql.swift.elements.stmt.ForEachStmt` and added a mention of the new predicates. I think that gets us out of needing a deprecation period?

Reply:
Yes, it's a good solution. :)